lib/setup-tofu.js

```diff
@@ -0,0 +1,170 @@
+  const mappings = {
+    win32: 'windows'
+  };
+  return mappings[os] || os;
+}
+
+async function downloadCLI (url) {
+  core.debug(`Downloading OpenTofu CLI from ${url}`);
+  const pathToCLIZip = await tc.downloadTool(url);
+
+  let pathToCLI = '';
+
+  core.debug('Extracting OpenTofu CLI zip file');
+  if (os.platform().startsWith('win')) {
+    core.debug(`OpenTofu CLI Download Path is ${pathToCLIZip}`);
+    const fixedPathToCLIZip = `${pathToCLIZip}.zip`;
+    io.mv(pathToCLIZip, fixedPathToCLIZip);
+    core.debug(`Moved download to ${fixedPathToCLIZip}`);
+    pathToCLI = await tc.extractZip(fixedPathToCLIZip);
+  } else {
+    pathToCLI = await tc.extractZip(pathToCLIZip);
+  }
+
+  core.debug(`OpenTofu CLI path is ${pathToCLI}.`);
+
+  if (!pathToCLIZip || !pathToCLI) {
+    throw new Error(`Unable to download OpenTofu from ${url}`);
+  }
+
+  return pathToCLI;
+}
+
+async function installWrapper (pathToCLI) {
+  let source, target;
+
+  // If we're on Windows, then the executable ends with .exe
+  const exeSuffix = os.platform().startsWith('win') ? '.exe' : '';
+
+  // Rename tofu(.exe) to tofu-bin(.exe)
+  try {
+    source = [pathToCLI, `tofu${exeSuffix}`].join(path.sep);
+    target = [pathToCLI, `tofu-bin${exeSuffix}`].join(path.sep);
+    core.debug(`Moving ${source} to ${target}.`);
+    await io.mv(source, target);
+  } catch (e) {
+    core.error(`Unable to move ${source} to ${target}.`);
+    throw e;
+  }
+
+  // Install our wrapper as tofu
+  try {
+    source = path.resolve([__dirname, '..', 'wrapper', 'dist', 'index.js'].join(path.sep));
+    target = [pathToCLI, 'tofu'].join(path.sep);
+    core.debug(`Copying ${source} to ${target}.`);
+    await io.cp(source, target);
+  } catch (e) {
+    core.error(`Unable to copy ${source} to ${target}.`);
+    throw e;
+  }
+
+  // Export a new environment variable, so our wrapper can locate the binary
+  core.exportVariable('TOFU_CLI_PATH', pathToCLI);
+}
+
+// Add credentials to CLI Configuration File
+// https://www.tofu.io/docs/commands/cli-config.html
+async function addCredentials (credentialsHostname, credentialsToken, osPlat) {
+  // format HCL block
+  // eslint-disable
+  const creds = `
+credentials "${credentialsHostname}" {
+  token = "${credentialsToken}"
+}`.trim();
+  // eslint-enable
+
+  // default to OS-specific path
+  let credsFile = osPlat === 'win32'
+    ? `${process.env.APPDATA}/terraform.rc`
+    : `${process.env.HOME}/.terraformrc`;
+
+  // override with TF_CLI_CONFIG_FILE environment variable
+  credsFile = process.env.TF_CLI_CONFIG_FILE ? process.env.TF_CLI_CONFIG_FILE : credsFile;
+
+  // get containing folder
+  const credsFolder = path.dirname(credsFile);
+
+  core.debug(`Creating ${credsFolder}`);
+  await io.mkdirP(credsFolder);
+
+  core.debug(`Adding credentials to ${credsFile}`);
+  await fs.writeFile(credsFile, creds);
+}
+
+async function run () {
+  try {
+    // Gather GitHub Actions inputs
+    const version = '1.6.0-alpha1';
```

Comment:
Why not dynamic? seems like you already have the get input line ready

Reply:
To release v0.0.1 and let people use it. The user's input can be treated included as feature of v0.0.2 of the action. Unfortunately I didn't have time to deliver everything at once.

Reply:
let's update for `alpha2` for now

Reply:
Done